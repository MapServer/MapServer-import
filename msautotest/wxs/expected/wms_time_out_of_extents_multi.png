<HTML>
<HEAD><TITLE>MapServer Message</TITLE></HEAD>
<!-- MapServer version 4.7 OUTPUT=GIF OUTPUT=PNG OUTPUT=JPEG OUTPUT=WBMP OUTPUT=SVG SUPPORTS=PROJ SUPPORTS=FREETYPE SUPPORTS=WMS_SERVER SUPPORTS=WMS_CLIENT SUPPORTS=WFS_SERVER SUPPORTS=WFS_CLIENT SUPPORTS=WCS_SERVER INPUT=EPPL7 INPUT=POSTGIS INPUT=OGR INPUT=GDAL INPUT=SHAPEFILE -->
<BODY BGCOLOR="#FFFFFF">
loadSymbolSet(): Unable to access file. (../etc/symbols.sym) <br>
</BODY></HTML>