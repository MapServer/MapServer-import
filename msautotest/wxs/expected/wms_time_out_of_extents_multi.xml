Content-type: application/vnd.ogc.se_xml

<?xml version='1.0' encoding="ISO-8859-1" standalone="no" ?>
<!DOCTYPE ServiceExceptionReport SYSTEM "http://ogc.dmsolutions.ca/wms/1.1.0/exception_1_1_0.dtd">
<ServiceExceptionReport version="1.1.0">
<ServiceException code="InvalidDimensionValue">
msWMSApplyTime: WMS server error. Time value(s) 2004-01-01 14:10:00,2004-12-12 14:10:00 given is invalid or outside the time extent defined (2004-01-01/2004-02-01).
</ServiceException>
</ServiceExceptionReport>
